analysis.go

```diff
@@ -112,33 +102,15 @@ func checkTasks(checkFactory getProjectCheckFactory, project string, outFor, err
 	}
 }
 
-// getResourceStruct retrieves the requested resource in the supplied project
-// from the platform and parse the JSON into the supplied interface.
-func getResourceStruct(project, resource string, dest interface{}) error {
-	stdOut := bytes.NewBuffer([]byte{})
-	stdErr := bytes.NewBuffer([]byte{})
-	outFor := func(project, resource string) (io.Writer, io.Closer, error) {
-		return stdOut, ioutil.NopCloser(nil), nil
-	}
-	errOutFor := func(project, resource string) (io.Writer, io.Closer, error) {
-		return stdErr, ioutil.NopCloser(nil), nil
-	}
-	task := ResourceDefinitions(project, []string{resource}, outFor, errOutFor)
-
-	err := task()
-	if err != nil {
-		return err
-	}
-
-	stdErrString := string(stdErr.Bytes())
-	if stdErrString != "" {
-		return errors.New(stdErrString)
-	}
+// getDumpedResourceAsStruct retrieves the requested resource from the dump directory and
+// decodes the JSON into the provided interface
+func getDumpedResource(path []string, dest interface{}) error {
+	filePath := filepath.Join(path...)
+	contents, err := ioutil.ReadFile(filePath)
 
-	decoder := json.NewDecoder(stdOut)
+	decoder := json.NewDecoder(bytes.NewBuffer(contents))
```

Comment:
This is begging to be `json.Unmarshal` instead; see https://golang.org/pkg/encoding/json/#Unmarshal.